app/code/community/Sailthru/Email/Model/Client/Content.php

```diff
@@ -55,6 +55,7 @@ public function saveProduct(Mage_Catalog_Model_Product $product)
      */
     public function getProductData(Mage_Catalog_Model_Product $product)
     {
+        $stockItem = Mage::getModel('cataloginventory/stock_item')->loadByProduct($product);
```

Comment:
Would this be inefficient on large batch product jobs? Maybe it could be a boolean (toggleable for use-caseslike the mass action which already gets it via join here: https://github.com/sailthru/sailthru-magento-extension/blob/develop/app/code/community/Sailthru/Email/controllers/ContentController.php#L50).